site/js/modules/webui/server.js

```diff
@@ -167,11 +167,27 @@ function renderEndpointArea(endpoints, selectedEndpoint, context) {
 function callEndpoint(targetEndpoint, inputFields, context, container) {
     Routing.loadingStart(container.querySelector(".results-area"), false);
 
+    let override_email = undefined;
+    let override_cleartext = undefined;
+
     let params = {};
+    let clearContextUser = true;
```

Comment:
For call-endpoint, wouldn't we just never clear the context user?
It seems a bit awkward if you click a button on the page and then you get logged out of the entire app.